quaint/src/connector/postgres/native/mod.rs

```diff
@@ -205,7 +205,7 @@ impl PostgresNativeUrl {
     pub(crate) fn to_config(&self) -> Config {
         let mut config = Config::new();
 
-        config.user(self.username().borrow());
+        config.user(self.username().borrow() as &str);
         config.password(self.password().borrow() as &str);
```

Comment:
Let's just replace it with

```suggestion
        config.user(self.username().as_ref());
        config.password(self.password().as_ref());
```

(or `&*self.username()`, either is fine with me) and remove `use std::borrow::Borrow`.

Using the `Borrow` trait here is weird in the first place, it's not really meant to be used outside of generic code.